src/core/string_set_test.cc

```diff
@@ -121,6 +121,58 @@ TEST_F(StringSetTest, StandardAddErase) {
   EXPECT_TRUE(ss_->Erase("AAAAAAAAAAAAAAA@"));
 }
 
+TEST_F(StringSetTest, DisplacedBug) {
+  string_view vals[] = {"imY", "OVl", "NhH", "BCe", "YDL", "lpb",
+                        "nhF", "xod", "zYR", "PSa", "hce", "cTR"};
+  ss_->AddMany(absl::MakeSpan(vals), UINT32_MAX);
+
+  ss_->Add("fIc");
```

Comment:
:scream:

Reply:
(that builder pattern would be :100: here -- it's fine we don't really needed just mentioning this)